tests/config.c

```diff
@@ -344,6 +366,13 @@ config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
                                (MAX_PAGES_PER_EXTENT * cfg[cfg_idx].page_size));
             return STATUS_BAD_PARAM;
          }
+         if ((cfg[cfg_idx].num_normal_bg_threads == 0)
```

Comment:
This check appears below on L384. I had checked that in as part of the previous rework to this file to support bg-threads.